### PR TITLE
Feat: Add llm rag guide

### PR DESCRIPTION
**Pull request by qq157755587**
### Description of your changes

LLM RAG guide

### CODE REVIEW

File: docs/zh/user-tutorials/llm-rag-guide.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+- jupyterlab
+
+ollama 用于在本地运行大模型，milvus 用于保存向量化之后的数据，而 jupyterlab 则是开发环境。
+
+### 在本地运行大模型
+
+我们以 [phi3](https://ollama.com/library/phi3) 为例运行大模型。进入 ollama 的容器，执行
+
+```shell
+ollama run phi3:3.8b
```
Comment (xingcan-ltc) [inline docs/zh/user-tutorials/llm-rag-guide.md:43]: ollama pull phi3:3.8b

File: docs/zh/user-tutorials/llm-rag-guide.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+- ollama
+- milvus
+- jupyterlab
+
+ollama 用于在本地运行大模型，milvus 用于保存向量化之后的数据，而 jupyterlab 则是开发环境。
+
+### 在本地运行大模型
+
+我们以 [phi3](https://ollama.com/library/phi3) 为例运行大模型。进入 ollama 的容器，执行
```
Comment (xingcan-ltc) [inline docs/zh/user-tutorials/llm-rag-guide.md:40]: `kubectl exec -it $(kubectl get pods -l app.kubernetes.io/name=ollama -n kdp-data -o jsonpath='{.items[0].metadata.name}') -n kdp-data -- ollama pull  phi3:3.8b`

File: docs/zh/user-tutorials/llm-rag-guide.md (hunk 1 above)
Comment (qq157755587) [reply docs/zh/user-tutorials/llm-rag-guide.md]: updated.

File: docs/zh/user-tutorials/llm-rag-guide.md (hunk 2 above)
Comment (qq157755587) [reply docs/zh/user-tutorials/llm-rag-guide.md]: updated.

**Review by jason-ltc — APPROVED** [review]

**Pull request merged**